tools/dev-upload.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 MIR_VERSION=$( dpkg-parsechangelog --show-field Version \
                | perl -n -e '/(^[^~-]+)/ && print $1' )
-GIT_COMMITS=$( git rev-list --count HEAD )
+GIT_COMMITS=$( git rev-list --count `git rev-parse --abbrev-ref HEAD@{u}` )
```

Comment:
This reads the commit count in the remote branch tracked by HEAD, regardless of what's checked out.

Reply:
Actually that's not true. It still only works on what's fetched. :/